ansible/module_utils/port_utils.py

```diff
@@ -437,6 +437,11 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
             # this device simulates 32 ports, with 4 as the step for port naming.
             for i in range(0, 32, 4):
                 port_alias_to_name_map["Ethernet%d" % i] = "Ethernet%d" % i
+        elif hwsku == "Cisco-8102-28FH-DPU-O-T1":
+            for i in range(0, 28, 8):
```

Comment:
the port list doesn't look right:

```
>>> [i for i in range(0, 28, 8)]
[0, 8, 16, 24]
```

Reply:
it should be 224 instead of 28, if I get it right.

Reply:
yes last port will be Ethernet216

Reply:
```
>>> for i in range(0, 217, 8): print("Ethernet%d" % i)
...
Ethernet0
Ethernet8
Ethernet16
Ethernet24
Ethernet32
Ethernet40
Ethernet48
Ethernet56
Ethernet64
Ethernet72
Ethernet80
Ethernet88
Ethernet96
Ethernet104
Ethernet112
Ethernet120
Ethernet128
Ethernet136
Ethernet144
Ethernet152
Ethernet160
Ethernet168
Ethernet176
Ethernet184
Ethernet192
Ethernet200
Ethernet208
Ethernet216
```